lib/components/Avatar.js

```diff
@@ -49,49 +70,56 @@ const Avatar = ({ contact, className, size, isOnline, activity }) => {
   const renderUserInitial = userName => {
     const userInitial = getInitials(userName);
     const charCode =
-      (userInitial ? userInitial.charCodeAt(0) : 0) +
-      (userInitial ? userInitial.charCodeAt(1) : 0);
+      (userInitial.charCodeAt(0) || 0) +
+      (userInitial.charCodeAt(1) || 0);
     const bgColor =
       tailwindColors[(charCode % 65) % tailwindColors.length] || "gray";
-    const backgroundClass = `bg-${bgColor}-400`;
+    const backgroundClass = `bg-${bgColor}-300`;
 
     return (
       <div
         className={classnames([
           "relative flex flex-row items-center justify-center",
-          "text-sm text-white font-medium rounded-full",
+          "text-white font-medium rounded-full",
           backgroundClass,
           className
         ])}
         style={style}
       >
         {userInitial}
-        {isOnline && <Indicator />}
+        <Indicator status={status} />
```

Comment:
We should have the `status` check. Else an unnecessary `<span>` will be rendered all the time right.